graylog2-server/src/main/java/org/graylog/plugins/views/search/rest/scriptingapi/request/MetricConfiguration.java

```diff
@@ -20,6 +20,7 @@
 
 public interface MetricConfiguration {
 
+    @Deprecated(forRemoval = true)
     default Optional<String> columnName(final Metric metric) {
```

Comment:
Is it still used anywhere after the change?

Reply:
Removed, thx!